roles/frr/templates/frr_loadbalancer_external_uplink.conf.j2

```diff
@@ -0,0 +1,63 @@
+frr version 8.1
+frr defaults traditional
+hostname {{ inventory_hostname_short }}
+log syslog informational
+service integrated-vtysh-config
+!
+router bgp {{ frr_local_as }}
+ no bgp ebgp-requires-policy
+ bgp bestpath as-path multipath-relax
+ bgp router-id {{ frr_loopback_v4 }}
+{% for item in frr_uplinks + frr_uplinks_external %}
+ neighbor {{ item.interface }} interface remote-as {{ item.remote_as }}
+{% endfor %}
+ !
+ address-family ipv4 unicast
+  redistribute connected
+{% for item in frr_uplinks %}
+  neighbor {{ item.interface }} route-map bgp_out_internal out
+{% endfor %}
+{% for item in frr_uplinks_external %}
+  neighbor {{ item.interface }} route-map bgp_in_external in
+  neighbor {{ item.interface }} route-map bgp_out_external out
+{% endfor %}
+  maximum-paths 2
+ exit-address-family
+ !
+ address-family ipv6 unicast
+  redistribute connected
+{% for item in frr_uplinks %}
+  neighbor {{ item.interface }} activate
+  neighbor {{ item.interface }} route-map bgp_out_internal out
+{% endfor %}
+{% for item in frr_uplinks_external %}
+  neighbor {{ item.interface }} activate
+  neighbor {{ item.interface }} route-map bgp_in_external in
+  neighbor {{ item.interface }} route-map bgp_out_external out
+{% endfor %}
+ exit-address-family
+exit
+!
+route-map bgp_out_internal permit 10
+ match interface {{ frr_dummy_interface_internal }}
+exit
+!
+route-map bgp_out_external permit 10
+ match interface {{ frr_dummy_interface_external }}
+exit
+!
+route-map bgp_in_external permit 10
+ set table {{ frr_incoming_table_external }}
```

Comment:
Add `frr_incoming_table_external` with a sane value to `defaults/main.yml`.

Reply:
Now defaulting to 254 (default according to https://man7.org/linux/man-pages/man8/ip-route.8.html#:~:text=By%20default%20all%20normal%20routes,local%20table%20(ID%20255). )